music21/abcFormat/__init__.py

```diff
@@ -1811,9 +1835,15 @@ def tokenize(self, strSrc):
         collect = []
         lastIndex = len(strSrc) - 1
         skipAhead = 0
+        accidentalsAndDecorations = '.~^=_HLMOPSTuv'
+        accidentals = '^=_'
 
         activeChordSymbol = '' # accumulate, then prepend
-
+        accidentalized = dict()
```

Comment:
as above

Reply:
should accidentalized be stored in `self.` so it can be used in sub functions?

Reply:
{} are used throughout to initialize dictionaries. Not sure about storing accidentalized in self. It really only has utility within tokenize().